src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Custody.java

```diff
@@ -40,4 +42,11 @@
     @OneToMany(cascade = {CascadeType.ALL}, orphanRemoval=true, mappedBy = "custody")
     private List<KeyDate> keyDates;
 
+    @OneToMany
+    @JoinColumn(name = "CUSTODY_ID")
+    private List<Release> releases;
+
+    public Optional<Release> findLatestRelease() {
+        return this.getReleases().stream().max(Comparator.comparing(Release::getActualReleaseDate));
```

Comment:
Maybe filter by `softDeleted` though not sure if if this is ever set by Delius

Reply:
I've dealt with the soft delete flag on the Release and Recall entities.  However I've left the custodial event / disposal / custody todo in the next piece of work as I will be refactoring some other stuff in that area.